bin/zuul

```diff
@@ -271,6 +271,11 @@ scout_browser(function(err, all_browsers) {
                     console.log('%s console'.white, name);
                 }
 
+                //When testing with microsoft edge:
+                //Adds length property to array-like object if not defined to execute console.log properly
+                if(msg.args.length === undefined){
```

Comment:
I did, that's why I put a space before the comment ;)
cf:
```js
 // configures sauce connect with a tunnel identifier
        // if sauce_connect is true, use the TRAVIS_JOB_NUMBER environment variable
        // otherwise use the contents of the sauce_connect variable
        if (self._opt.sauce_connect) {
```